www/js/lib/popovers.js

```diff
@@ -104,14 +104,12 @@ function cleanUpLedeContent (node) {
     styleElements.forEach(style => {
         style.parentNode.removeChild(style);
     });
-    const paragraphs = Array.from(node.querySelectorAll('p'));
-    // Filter out empty paragraphs or those with less than 50 characters
```

Comment:
This comment shouldn't be deleted, but it should be moved above line 108 (above line beginning `const parasWithContent = ...`.

Reply:
Got it , Fixing this.